libiocage/lib/events.py

```diff
@@ -15,7 +17,7 @@ class IocageEvent:
     Base class for all other iocage events
     """
 
-    HISTORY = []
+    HISTORY: List[Any] = []
```

Comment:
this is not ideal, but i have idea how to tell it the type of this class

Reply:
Guess I can explain that: `List[libiocage.lib.events.IocageEvent]`

The class attribute, not the instance attribute contains a list of all events occurred. This way every event has access to the full history.

Reply:
python (or mypy) can't find the type `libiocage.lib.events.IocageEvent`

Reply:
Within this file it's just `IocageEvent`, not the fqdn :trollface:

Reply:
yeah, that's what I thought too, but that doesn't work either.